src/Service/CredentialsRevoker/DoctrineCredentialsRevoker.php

```diff
@@ -83,15 +84,15 @@ public function revokeCredentialsForClient(Client $client): void
                     ->where('at.client = :client')
                     ->getDQL()
             ))
-            ->setParameter('client', $doctrineClient)
+            ->setParameter('client', $doctrineClient->getIdentifier(), 'string')
             ->getQuery()
             ->execute();
 
         $this->entityManager->createQueryBuilder()
             ->update(AuthorizationCode::class, 'ac')
             ->set('ac.revoked', true)
             ->where('ac.client = :client')
-            ->setParameter('client', $doctrineClient)
+            ->setParameter('client', $doctrineClient->getIdentifier(), 'string')
```

Comment:
this is a performance enhancement ( detected by psalm symfony plugin - as doctrine has to infer the type later on by it self ), i didn't use `Type::STRING` or `Types::STRING` as `Type` class is deprecated when using highest set of dependencies where we run psalm - resulting in an error - and `Types` doesn't exist when running tests with lowest set of dependencies, so it fails.